lib/internal/util.js

```diff
@@ -201,17 +201,19 @@ function getConstructorOf(obj) {
 const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
 const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
 
-function promisify(orig) {
-  if (typeof orig !== 'function') {
+function promisify(original) {
```

Comment:
I had another look at it and I changed it to original because [the error](https://github.com/nodejs/node/pull/13829/files#diff-a43208147d795be6dd3517c53226e37dR207) is thrown with `original` and not `orig`. I originally changed the error but @refack suggested to change the variable name instead as the documentation also speaks of `original`.